<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Harmeet-PC" ignored="0" name="TestRetry" tests="9" failures="2" timestamp="25 Feb 2020 14:06:48 GMT" time="0.161" errors="0">
  <testcase name="RetryTC1" time="0.024" classname="com.pcs.retryAnalyzer.NewTest">
    <skipped/>
  </testcase> <!-- RetryTC1 -->
  <testcase name="RetryTC1" time="0.007" classname="com.pcs.retryAnalyzer.NewTest">
    <skipped/>
  </testcase> <!-- RetryTC1 -->
  <testcase name="RetryTC1" time="0.004" classname="com.pcs.retryAnalyzer.NewTest">
    <skipped/>
  </testcase> <!-- RetryTC1 -->
  <testcase name="RetryTC1" time="0.002" classname="com.pcs.retryAnalyzer.NewTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.pcs.retryAnalyzer.NewTest.RetryTC1(NewTest.java:11)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- RetryTC1 -->
  <testcase name="RetryTC2" time="0.007" classname="com.pcs.retryAnalyzer.NewTest">
    <skipped/>
  </testcase> <!-- RetryTC2 -->
  <testcase name="RetryTC2" time="0.004" classname="com.pcs.retryAnalyzer.NewTest">
    <skipped/>
  </testcase> <!-- RetryTC2 -->
  <testcase name="RetryTC2" time="0.002" classname="com.pcs.retryAnalyzer.NewTest">
    <skipped/>
  </testcase> <!-- RetryTC2 -->
  <testcase name="RetryTC2" time="0.003" classname="com.pcs.retryAnalyzer.NewTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.pcs.retryAnalyzer.NewTest.RetryTC2(NewTest.java:16)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- RetryTC2 -->
  <testcase name="RetryTC3" time="0.002" classname="com.pcs.retryAnalyzer.NewTest"/>
</testsuite> <!-- TestRetry -->
